src/analysis/inner_product_space/basic.lean

```diff
@@ -860,12 +860,17 @@ lemma norm_eq_sqrt_real_inner (x : F) : ∥x∥ = sqrt ⟪x, x⟫_ℝ :=
 by { have h := @norm_eq_sqrt_inner ℝ F _ _ x, simpa using h }
 
 lemma inner_self_eq_norm_sq (x : E) : re ⟪x, x⟫ = ∥x∥ * ∥x∥ :=
```

Comment:
Should we consider renaming this:
```suggestion
lemma inner_self_eq_norm_mul_norm (x : E) : re ⟪x, x⟫ = ∥x∥ * ∥x∥ :=
```
to free up `inner_self_eq_norm_sq` for the new lemma you're adding?

(and likewise for `real_inner_self_eq_norm_sq` ofc)